packages/cardano-services/src/Program/loadHttpServer.ts

```diff
@@ -65,7 +67,10 @@ const serviceMapFactory = (args: ProgramArgs, logger: Logger, db?: Pool) => ({
 
     return await NetworkInfoHttpService.create({
       logger,
-      networkInfoProvider: new DbSyncNetworkInfoProvider(args.options?.cardanoNodeConfigPath, db, logger)
+      networkInfoProvider: new DbSyncNetworkInfoProvider(
+        { cardanoNodeConfigPath: args.options?.cardanoNodeConfigPath, pollInterval: POLL_INTERVAL },
```

Comment:
`POLL_INTERVAL` could be passed as an env var as well, any thoughts?

Reply:
Agree, maybe `CACHE_POLL_INTERVAL`?

Reply:
Actual the poll interval is not associated with the cache, rather than with the DB sync provider. The cache service shouldn't know about the provider's custom polling, no? 🤔

Reply:
Right. You don't have to use this exact name, just wanted to point out that POLL_INTERVAL is too generic for an env variable

Reply:
`DB_POLL_INTERVAL` ?

Reply:
[6e5ee7c](https://github.com/input-output-hk/cardano-js-sdk/pull/276/commits/6e5ee7c71babfe32d15e36f92ea25177a42f75a5)